packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue

```diff
@@ -77,6 +78,11 @@ export default defineComponent<KottiUserMenu.PropsInternal>({
 			clickawayMenu: () => (isMenuShow.value = false),
 			isMenuShow,
 			isNarrow,
+			parsedSections: computed(() =>
+				props.sections
```

Comment:
Why the ternary here?

Reply:
I removed it now. The reason for this is a VSCode bbug (maybe?) that confused me about the type
<img width="601" alt="Screenshot 2021-11-25 at 14 22 00" src="https://user-images.githubusercontent.com/16950410/143449225-4fbf2d0e-a1f4-4a72-b1d1-4678ea4ae5c6.png">
<img width="874" alt="Screenshot 2021-11-25 at 14 22 14" src="https://user-images.githubusercontent.com/16950410/143449233-bf65f5d8-2d6c-4615-9b14-b8811dad2ecb.png">

Reply:
Yeah, it’s a visual bug in vetur with `makeProps`. It somehow shows-up as optional even though it’s not. Cause is unknown but doesn’t seem to affect anything besides showing this weirdly